airflow/cli/commands/provider_command.py

```diff
@@ -40,17 +43,20 @@ def _tabulate_providers(providers: List[Dict], tablefmt: str):
     return tabulate(tabulate_data, tablefmt=tablefmt, headers='keys')
 
 
+def _remove_rst_syntax(value: str) -> str:
+    return re.sub("[`_<>]", "", value.strip(" \n."))
+
+
 def provider_get(args):
     """Get a provider info."""
     providers = ProvidersManager().providers
     if args.provider_name in providers:
         provider_version = providers[args.provider_name][0]
         provider_info = providers[args.provider_name][1]
-        print("#")
-        print(f"# Provider: {args.provider_name}")
-        print(f"# Version: {provider_version}")
-        print("#")
+        print(f"Provider: {args.provider_name}")
+        print(f"Version: {provider_version}")
         if args.full:
+            provider_info["description"] = _remove_rst_syntax(provider_info["description"])
```

Comment:
I would say that if `args.full` is passed we should not print the first two lines - just output the yaml / json, this will allow users to use it in `jq` for example